index.html

```diff
@@ -279,7 +279,87 @@ <h2 class="about-me-heading">About me</h2>
           <li>Terminal</li>
         </ul>
       </div>
+
+      <!--CONTACTS-->
+      <div class="contact-info">
+        <p class="contact-text">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </p>
+
+        <!--contact form-->
+        <form action="https://formspree.io/f/xlezlyqd" method="POST">
+          <div class="form-group">
+            <input
+              type="text"
+              name="name"
+              id="name"
+              class="input-style"
+              maxlength="30"
+              placeholder="Full name"
+              required
+            />
+          </div>
+
+          <div class="form-group">
+            <input
+              type="email"
+              name="email"
+              id="email"
+              class="input-style"
+              placeholder="Email address"
+              required
+            />
+          </div>
+
+          <div class="form-group">
+            <textarea
+              id="comments"
+              class="input-textarea"
+              name="comment"
+              maxlength="500"
+              placeholder="Enter message here"
+              required
+            ></textarea>
+          </div>
+
+          <a id="submit" class="submit-button" href="#">Get in touch</a>
+        </form>
```

Comment:
- [ ] Please also ensure to include styles for all `button` state interactions which are `hover`, `enabled`, `pressed` and `disabled`. Reference the below image

![image](https://user-images.githubusercontent.com/43995830/159667370-7c58a1b6-6eff-4302-9427-18b708182c95.png)

Reply:
Fixed. Added button effects